frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx

```diff
@@ -13,13 +15,12 @@ export function Card(props) {
                 <h3 className={style['card-detail']}>{props.content.contact}</h3>
                 <p>{props.content.description}</p>
                 <p>{props.content.email}</p>
-                <a className={style['card-link']} href={props.content.linkedin}>Linkdin</a>
-                 <div className={style["button-group"]}>
-                    <button className={style["button-edit"]}>URL</button>
-                    <button className={style["button-delete"]}>Delete</button>
+                <a className={style['card-link']} href={props.content.linkdin} target="_blank" rel="noreferrer">LinkedIn</a>
+                 <div className={style["button-group"]}>  
+                    <button className={style["button-delete"]}   onClick={() => props.OnClickDelete(props.content._id)}>Delete</button>
                   </div>
             </div>
-
+          
```

Comment:
[neat] please make sure the formatting is consistent, i see few extra space here which is un-necessary